tests/test_vqe.py

```diff
@@ -487,6 +487,7 @@ def test_arithmetic_errors(self):
             H -= A
 
 
+@pytest.mark.usefixtures("tape_mode")
```

Comment:
After fixing `qml.map`, `VQECost` tests pass with minor test modification, and no source code changes 😄